literature.bib

```diff
@@ -19077,6 +19264,15 @@ @book{L09
 	publisher= {"O'Reilly Media, Inc."}
 }
 
+@article{LUV:ICSOFT09,
+	title={{Software Product Line Testing}},
+	author={Lamancha, Beatriz P{\'e}rez and Usaola, Macario Polo and Velthius, Mario Piattini},
+	journal={A Systematic Review. ICSOFT (1)},
```

Comment:
Does not look right

Reply:
yes, something seems to be off
see here: https://dblp.org/rec/conf/icsoft/LamanchaPP10.html?view=bibtex